api/client/gro_client.py

```diff
@@ -62,9 +62,12 @@ def get_df(self, show_revisions=False, index_by_series=False):
                 data_series['show_revisions'] = True
             self.add_points_to_df(None, data_series, self.get_data_points(**data_series))
         if index_by_series:
-            return self._data_frame.set_index([c for c in filter(
```

Comment:
hmm filter is more readabe, why change this?

Reply:
Controversial opinion, but filter+lambda vs list comprehension debate aside:

In this case it was already using a list comprehension to begin with. So, going from 3 elements (comprehension+filter+lambda) down to 1 (just list comprehension) seems like the less complex version to me

Reply:
yes  should just be `list(filter(...))` rather than hybrid.  computationally both are trivial but readability-wise i've settled on  filter() as the common pattern for this kid of stuff here (i'm sure some inconsistencies  here and there but let's reduce them).

Reply:
Okay, changed back to the `list(filter(lambda...))` syntax and moved into a named `intersect` function